docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+
+**Vulnerability metadata** 
+
+Clair depends on the vulnerability metadata to complete the analysis process. After the first initial installation, Clair will automatically start to update the metadata database from different vulnerability repositories. The updating process might be take a while based on the data size and network situations etc.. If the database is not fully ready, there will be a warning tips at the footer of the repositories datagrid view.
+![browse project](img/clair_not_ready.png)
+
+The 'database not fully ready' warning message is also displayed in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'** for your awareness.
+![browse project](img/clair_not_ready2.png)
+
+Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. Click the overall timestamp above, the detailed timestamps list for different namespaces are dropped down.
+![browse project](img/clair_ready.png)
+
+**Scan Image** 
+
+Enter your project and locate the specified repository. Expand the tag list via clicking the arrow icon on the left side. For each tag there will be an 'Vulnerability' column to display vulnerability scanning status and results related information. You can click the vertical ellipsis to open the popup menu and then click **'Scan'** to start the vulnerability analysis process. 
+![browse project](img/scan_menu_item.png)
+**NOTES: Only the users with 'Project Admin' role have the accesses to trigger the analysis process.**
+
+The analysis process may have the following status that are indicated in the 'Vulnerability' column:
+* **Not Scanned:** The tag is never scanned so far.
+* **Queued:** The task is scheduled but not executed yet.
+* **Scanning:** The process is in progress.
+* **Error:** The process is failed to complete.
+* **Complete:** The process is successfully done.
+
+For the **'Not Scanned'** and **'Queued'** statuses, a text label with status information is shown. For the **'Scanning'**, a progress bar will be displayed.
+If error occurred, you can click the **'View Log'** link to open the log viewer to check the related logs.
+![browse project](img/log_viewer.png)
+
+If process is successfully completed, a result bar chart is created. The width of the different colors are used to indicate the percent of features with different severity vulnerabilities found in the overall features. 
+* **Color Red:** High level vulnerabilities
+* **Color Orange:** Medium level vulnerabilities
+* **Color Yellow:** Low level vulnerabilities
+* **Color Grey:** Unknown level vulnerabilities
+* **Color Green:** None vulnerabilities

```

Comment:
* **Red:** High level of vulnerabilities
* **Orange:** Medium level of vulnerabilities
* **Yellow:** Low level of vulnerabilities
* **Grey:** Unknown level of vulnerabilities
* **Green:** No vulnerabilities